integration_tests/setup_test.go

```diff
@@ -173,16 +176,26 @@ func (s *IntegrationTestSuite) initNodesWithMnemonics(mnemonics ...string) {
 
 	//initialize a genesis file for the first validator
 	val0ConfigDir := s.chain.validators[0].configDir()
-	for _, val := range s.chain.validators {
+	for i, val := range s.chain.validators {
+		// Fund the first validator with some funds to be used by auction module integration tests
+		balanceStr := initBalanceStr
+		if i == 0 {
+			balanceStr += ",100000000000gravity0x3506424f91fd33084466f402d5d97f05f8e3b4af"
+		}
 		s.Require().NoError(
-			addGenesisAccount(val0ConfigDir, "", initBalanceStr, val.keyInfo.GetAddress()),
+			addGenesisAccount(val0ConfigDir, "", balanceStr, val.keyInfo.GetAddress()),
 		)
 	}
 
 	// add orchestrator accounts to genesis file
-	for _, orch := range s.chain.orchestrators {
+	for i, orch := range s.chain.orchestrators {
+		// Fund the first orchestrator with some funds to be used by auction module integration tests
+		balanceStr := initBalanceStr
+		if i == 0 {
+			balanceStr += ",100000000000usomm"
```

Comment:
We may want to update the integration tests to use usomm as the authoritative token rather than a "test token".

Reply:
Yeah that's fair. I think it's kind of out of scope for this particular PR but I've opened up a follow up tracking ticket https://github.com/PeggyJV/sommelier/issues/191. It'll probably be cleaner to make this change in a dedicated PR.

Reply:
Sure, can address later.